kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -110,6 +109,14 @@
 
     private boolean findDeserializerFirst = Boolean.parseBoolean(System.getProperty("org.kie.server.json.findDeserializerFirst.enabled", "true"));
 
+    public static enum CNFEBehavior {
+        IGNORE,
+        WARN,
+        EXCEPTION
+    }
+
+    private CNFEBehavior customObjectDeserializerCNFEBehavior = CNFEBehavior.valueOf(System.getProperty(KieServerConstants.JSON_CUSTOM_OBJECT_DESERIALIZER_CNFE_BEHAVIOR, CNFEBehavior.IGNORE.name()));
```

Comment:
Could you handle then case when user provides invalid system property value? Currently the produced error is not much descriptive.

Reply:
@sutaakar Thanks, added.